app/controllers/api/policies_controller.rb

```diff
@@ -27,11 +27,11 @@ def edit_resource(type, id = nil, data = {})
     private
 
     def create_policy(data)
-      policy = MiqPolicy.create!(:name        => data.delete("name"),
-                                 :description => data.delete("description"),
-                                 :towhat      => data.delete("towhat"),
-                                 :mode        => data.delete("mode"),
-                                 :active      => true)
+      policy = MiqPolicy.create!(:name              => data.delete("name"),
+                                 :description       => data.delete("description"),
+                                 :target_class_name => data.delete("target_class_name"),
```

Comment:
Do we need to allow consumers to specify either one for some time?
```
:target_class_name => data.delete("target_class_name") || data.delete("towhat")
```

Reply:
👍 otherwise current API callers will HAVE to change their code when this thing ships.  Maybe also log a deprecation warning if we get a towhat for now.

Reply:
Good eye.  Yeah, I'm just at a point of getting tests to pass in all the repos.

Reply:
I'll mark them WIP for the purposes of getting cross repo passing